### PR TITLE
feat: [Breadcrumb] add a11y aria-*

### PR DESCRIPTION
**Pull request by boomboomchen**
[中文模板 / Chinese Template](https://github.com/DouyinFE/semi-design/blob/main/.github/PULL_REQUEST_TEMPLATE.zh-CN.md)

- [ ] I have read and followed [Pull Request Guidelines](https://github.com/DouyinFE/semi-design/blob/main/CONTRIBUTING-en-US.md#pull-request-guidelines) of the contributing guide.

### What kind of change does this PR introduce? (check at least one)

 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update
 - [ ] Refactor
 - [ ] Test Case
 - [ ] TypeScript definition update
 - [ ] Document improve
 - [ ] CI/CD improve
 - [ ] Branch sync
 - [ ] Other, please describe:

### PR description

Fixes #

### Changelog
🇨🇳 Chinese
- feat: breadcrumb 支持 a11y 语义化

---

🇺🇸 English
- feat: breadcrumb support a11y aria

### Checklist
- [ ] Test or no need
- [ ] Document or no need
- [ ] Changelog or no need

### CODE REVIEW

File: content/navigation/breadcrumb/index-en-US.md [hunk 1]
```diff
@@ -334,6 +334,10 @@ import { IconHome, IconArticle } from '@douyinfe/semi-icons';
 
 After **v>=1.16.0**, other props in Breadcrumb.Item are also supported correspondingly.
 
+## Accessibility
+
+- Breadcrumb supports the `aria-label` props to indicate the function of the Breadcrumb
```
Comment (shijiatongxue) [inline content/navigation/breadcrumb/index-en-US.md:339]: 面包屑的 aria-current 有点问题。

- [W3C 参考示例](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html)

image


image

- 我们

image

这里应该是之前就写的有问题。可以改下。

Comment (shijiatongxue) [reply content/navigation/breadcrumb/index-en-US.md:339]: 改完之后可以在文档这里写下。

File: packages/semi-ui/breadcrumb/index.tsx [hunk 2]
```diff
@@ -282,7 +288,7 @@ class Breadcrumb extends BaseComponent<BreadcrumbProps, BreadcrumbState> {
                     separator,
                 }}
             >
-                <nav aria-label={`${clsPrefix}`} className={sizeCls} style={style}>
+                <nav aria-label={this.props['aria-label'] || `${clsPrefix}`} className={sizeCls} style={style}>
```
Comment (shijiatongxue) [inline packages/semi-ui/breadcrumb/index.tsx:291]: 这里可以给个默认值，”Breadcrumb“

File: packages/semi-ui/breadcrumb/index.tsx [hunk 3]
```diff
@@ -41,6 +41,7 @@ export interface BreadcrumbProps extends BaseProps {
     renderMore?: (restItem: Array<React.ReactNode>) => React.ReactNode;
     /* Style type for ellipsis area */
     moreType?: MoreType;
+    'aria-label'?: string;
```
Comment (shijiatongxue) [inline packages/semi-ui/breadcrumb/index.tsx:44]: 改成 React.AriaAttributes['aria-label'] 吧

Comment (boomboomchen) [reply packages/semi-ui/breadcrumb/index.tsx:44]: 好的

**Pull request merged**